### PR TITLE
Update README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by BobVarioa**
Title. I tried to keep the style as close as possible, but let me know if there's any wording you'd like to change ~~(or any typos knowing myself 💀)~~.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,16 +264,16 @@ Porffor can run Test262 via some hacks/transforms which remove unsupported featu
 - `test262`: test262 runner and utils
 
 ## Usecases
-Basically none right now (other than giving people headaches). Potential ideas:
+Currently, Porffor is seriously limited in features and functionality, however it has some key benefits:
 - Safety. As Porffor is written in JS, a memory-safe language\*, and compiles JS to Wasm, a fully sandboxed environment\*, it is quite safe. (\* These rely on the underlying implementations being secure. You could also run Wasm, or even Porffor itself, with an interpreter instead of a JIT for bonus security points too.)
 - Compiling JS to native binaries. This is still very early!
+- Inline ASM for when you want to beat the compiler in performance, or just want fine grained functionality.
+- SIMD operations
```
Comment (CanadaHonk) [inline README.md:271]: Specify this is a future thing?

File: README.md [hunk 2]
```diff
@@ -249,16 +264,16 @@ Porffor can run Test262 via some hacks/transforms which remove unsupported featu
 - `test262`: test262 runner and utils
 
 ## Usecases
-Basically none right now (other than giving people headaches). Potential ideas:
+Currently, Porffor is seriously limited in features and functionality, however it has some key benefits:
 - Safety. As Porffor is written in JS, a memory-safe language\*, and compiles JS to Wasm, a fully sandboxed environment\*, it is quite safe. (\* These rely on the underlying implementations being secure. You could also run Wasm, or even Porffor itself, with an interpreter instead of a JIT for bonus security points too.)
 - Compiling JS to native binaries. This is still very early!
+- Inline ASM for when you want to beat the compiler in performance, or just want fine grained functionality.
```
Comment (CanadaHonk) [inline README.md:270]: `ASM` -> `Wasm`

File: README.md (hunk 1 above)
Comment (BobVarioa) [reply README.md:271]: Isn't there already some basic SIMD support? I remember seeing something in `bench`, has that been removed?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -209,6 +208,141 @@ Store the character code into the `out` pointer variable, and increment it.
 
 <br>
 
+### Porffor.wasm
+This is a macro that is essentially equivalent to c's `asm` macro. It allows you to write inline wasm bytecode in a similar format to [WAT](https://developer.mozilla.org/en-US/docs/WebAssembly/Understanding_the_text_format).
```
Comment (CanadaHonk) [inline CONTRIBUTING.md:212]: nit: `c's` -> `C's`, `wasm` -> `Wasm`

**Review by CanadaHonk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BobVarioa) [reply README.md:271]: Oh wait, it looks like it, sorry about that. I'll change that.

**Review by CanadaHonk — APPROVED** [review]
LGTM ty!

**Pull request merged**